src/page-overview/assets/tooltip.js

```diff
@@ -0,0 +1,19 @@
+import { getStateName } from "./data-forms";
+
+export default function initTooltips() {
+  const $tooltip = $("#map_tooltip");
+
+  $("#us_map .state")
+    .mousemove(function (event) {
+      $tooltip
+        .html(getStateName(event.target.id))
+        .css({
+          left: event.pageX + 15,
+          top: event.pageY + 15,
+        })
+        .removeClass("hidden");
+    })
+    .mouseout(function () {
+      $tooltip.html("").addClass("hidden");
+    });
```

Comment:
Is this robust?

Reply:
not sure what you mean by "robust" in this context? could you be more specific?